.github/automation/clang-format.sh

```diff
@@ -26,7 +27,18 @@ fi
 
 echo "Starting format check..."
 
-for filename in $(find "$(pwd)" -type f | grep -P ".*\.(c|cpp|h|hpp|cl)$"); do ${CLANG_FORMAT} -style=file -i $filename; done
+src_regex='.*\.(c|h|cpp|hpp|cxx|hxx|cl)$'
```

Comment:
The patch is ready for review. One thing to note is that I have added `.hxx|.cxx` to the regex pattern. This means some [files which were not checked before](https://github.com/uxlfoundation/oneDNN/blob/c8cd40f2502b99092f8d0dd60412516b816518a8/src/gpu/intel/gemm/jit/generator/pieces/atomic_fusions.cxx), might be now.

It might be a good idea to standardise `C` extensions to `.c|.h` and `C++` to `.cpp|.hpp` to reduce complexity for workflows like this, `clang-tidy`, etc.

Reply:
@atkassen, ah that makes sense, thanks. Is there any reason for the different file extensions?